website/redirects.txt

```diff
@@ -69,7 +69,7 @@
 /docs/http/sys-raw.html                /api/system/raw.html
 /docs/http/sys-health.html             /api/system/health.html
 
-/docs/guides/generate-root.html        /guides/configuration/generate-root.html
```

Comment:
All of these changes need redirects or else we'll break existing links and break SEO

Reply:
I added the missing redirects.  Thank you for catching this.